doc/pages/user/050-query_device.md

```diff
@@ -76,6 +79,13 @@ _Sample Response_
 
 This returns the Interfaces which the device reported in its Introspection *and* which are known to the Realm.
 
+Arbitrary information can be added to the device by means of `metadata`: they allow to store any number of string values associated to a corresponding string key.
+To set, modify and delete `metadata`, a `PATCH` on the device endpoint is required:
+```
+PATCH appengine.api.<your astarte domain>/v1/test/devices/f0VMRgIBAQAAAAAAAAAAAA
+```
+In the request body, the `data` JSON object should have a `metadata` key which bears a dictionary of strings. A valid request body which changes only device metadata, for example, is `{"data":{"metadata": {"<key>": "<value>"}}}`. Each key is unique and each time a metadata key is associated to a value, that value replaces the last stored value. To delete a metadata entry, setting the value to `null` is sufficient.
```

Comment:
change last sentence to:

To delete a metadata entry, set the value of the corresponding key to `null`. For example, POSTing `{"data":{"metadata": {"my_key": null}}}` will remove the `my_key` metadata entry from the device.